pkgs/servers/home-assistant/build-custom-component/check_manifest.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import json
+import importlib_metadata
+import sys
+
+from packaging.requirements import Requirement
```

Comment:
random off topic thought: I looked into modernizing pkgs/development/interpreters/python/catch_conflicts/catch_conflicts.py with importlib_metadata but the functions we are using were internal only...

Reply:
646c23a2f7711690cce032233be54dcb1a037965